apps/core/scenes/results/Results.js

```diff
@@ -76,7 +76,13 @@ class Results extends React.Component<Props> {
     props: ReturnResultsQueryResponse | OneWayResultsQueryResponse,
   ) => {
     const { searchData } = props;
-    return <ResultsList data={searchData} onBookPress={this.handleBookPress} />;
+    return this.props.layout < LAYOUT.largeMobile ? (
+      <View>
+        <ResultsList data={searchData} onBookPress={this.handleBookPress} />
+      </View>
```

Comment:
I'm not sure about this "hack". It works, but I'm slightly worried that it is not that readable.
By this we are just using default `View` styles to prevent results from scrolling.
Cannot we use same condition to simply disable `flex: 1` style on `resultContainer` which will have a similar effect and will be more obvious?

Reply:
@JoseDuSV Done!